cft_sidecar.yaml

```diff
@@ -521,7 +623,7 @@ Resources:
                         fi
                     fi
                     access_token=$(echo "$token" | jq -r .access_token)
-                    url="https://${ControlPlane}/deploy/docker-compose?TemplateVersion=$SIDECAR_VERSION"
+                    url="https://${ControlPlane}/deploy/docker-compose?TemplateType=cloudformation&TemplateVersion=$SIDECAR_VERSION"
```

Comment:
This `TemplateType` parameter is not necessary anymore in this call. The Terraform script uses it for backwards compatibility reasons.

Reply:
Actually, it is needed. If you don't supply a `TemplateType` set to either `cloudformation` or `terraform`, then the delivered YAML file is missing the following argument for dispatcher
```yaml
--load-balancer-tls-ports "${LOAD_BALANCER_TLS_PORTS}"
```